.changeset/eleven-geese-agree.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-app-taxes": minor
+---
+
+Added "document recording" toggle. It is turned on by default. When turned off, the document type of all Avatax transactions change to "SalesOrder", making them not record.
```

Comment:
Please add link to the changelog
https://developer.avalara.com/erp-integration-guide/sales-tax-badge/design-document-workflow/option-to-disable-document-recording/